contrib/completions.fish

```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
```

Comment:
Whoops, I think this one was added automatically...

Reply:
heh, it was at the very top of the file so I didn't notice. I'll fix it

Reply:
Thanks!